permissions/Email.permission

```diff
@@ -19,7 +19,8 @@ mkdir     ${HOME}/Downloads/mail_attachments
 whitelist ${HOME}/Downloads/mail_attachments
 
 # MessageServer socket
-whitelist /tmp/qcop-server-0
+noblacklist ${RUNUSER}/messageserver
```

Comment:
Can you noblacklist just a particular `${RUNUSER}/messageserver/qcop-server-0` but not the whole directory?

Reply:
I think I can't. blacklist rule prevents me from accessing that directory, noblacklist will prevent that rule, so if I say
```
blacklist ${RUNUSER}/messageserver
noblacklist ${RUNUSER}/messageserver/qcop-server-0
```
Then I don't get access to that messageserver directory still which means the files inside are not accessible either.

This directory was added for the socket so this noblacklist rule doesn't allow access to anything else than the socket.

Reply:
Ok, just has worries that something evil can deliberately be landed in this directory.

Reply:
We need to anyway restrict further this ${RUNUSER} access (change from blacklist rules to whitelist rules), there is separate task for that.